echopype/convert/parse_base.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, file, params, storage_options):
         self.config_datagram = None
         self.ping_data_dict = defaultdict(lambda: defaultdict(list))
         self.ping_time = defaultdict(list)  # store ping time according to channel
-        self.num_range_bin_groups = None  # number of range_bin groups
+        self.num_range_sample_groups = None  # number of range_sample groups
```

Comment:
This is not a require change, but a note that this is unused and will be removed.
I've noted it here: https://github.com/OSOceanAcoustics/echopype/issues/506#issuecomment-1079235949